### DISCUSSION TITLE
No devices in Homebridge

### DISCUSSION THREAD

**Discussion by mathewsears** [aarons22/homebridge-bond#136] [Q&A] [answered]
Not sure what exactly happened but I lost the ability to control my ceiling fans using homebridge. I’ve uninstalled and reinstalled the plugin and now have no ceiling fans in the home app or homebridge. The homebridge log does not seem to have much in the way of helpful information. Any ideas?

**Comment by aarons22** [comment, marked as answer]
Can you post your logs? Preferably in debug mode

**Reply by mathewsears** [reply]
This issue can be closed. I was able to get my devices working by updating packages on my raspbian.
